C++/Navio/Common/RCOutput.h

```diff
@@ -0,0 +1,17 @@
+#ifndef RCOUTPUT
+#define RCOUTPUT
+
+#define PWN_OUTPUT 0
```

Comment:
typo
s/PWN_OUTPUT/PWM_OUTPUT

And also it should be here

Reply:
Please, move it where it belongs. Maybe Examples? But most definitely not library code